python/cudf/cudf/_libxx/join.pxd

```diff
@@ -0,0 +1,28 @@
+from libcpp.vector cimport vector
```

Comment:
There's some discussion about moving `pxd` files under an `include` directory in PR https://github.com/rapidsai/cudf/pull/4045. It might be good to follow up with @shwina on if you should follow that.

Reply:
Please follow it, hope to have #4045 merged tonight :)

Reply:
Roger that - going to wait until that's merged to avoid a merge conflict.

Reply:
done.